### DISCUSSION TITLE
Does Bref work with Serverless's dashboard?

### DISCUSSION THREAD

**Discussion by brianwong3** [brefphp/bref#866] [Support]
Hi..Just discovered it is possible to run PHP on AWS Lambda which is a huge savior for me..or I will have to port my whole existing application to Typescript..😖

My only question is, I noticed that the Serverless dashboard doesn't get updated with the calls and status code metrics..Is this normal?

**Comment by mnapoli** [comment]
Hi, last time I checked the serverless dashboard did not support custom runtimes (which is what Bref uses), so no.

I personally use https://seed.run/ for some projects, it has a little dashboard too, but what I like the most about it is the CI/CD.

And this is also a reason why I'm working on a Bref Dashboard ^^ Though instead of an online SaaS, it's an application. You can download a preview version here if you want to try it out: https://download.dashboard.bref.sh/
